conf/containers.yml

```diff
@@ -36,10 +39,14 @@
     - [ "ssl/", "${conf.ssl_prefix}"]
     - [ "sync/", "${conf.sync_web_dir}" ]
   depend: [ "${conf.hostname_api}" ]
+  reload:
+    # TODO: do we need other directories synced?
+    - [ "${conf.dir_web}/app", "/usr/src/promis/app" ]
```

Comment:
Add stuff here to sync other folders as well. Avoid syncing the whole frontend directory since it will overmount `node_modules` on the container.